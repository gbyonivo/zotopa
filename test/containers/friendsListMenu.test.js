import React from 'react';
import renderer from 'react-test-renderer';
import { createStore } from 'redux';
import { Provider } from 'react-redux';
import combineReducers from '../../src/reducers';
import FriendListMenu from '../../src/containers/friendsListMenu';

const props = {
  fetchFriends: () => {}
};

const transactionReducer = {
  isFetchingFriends: false,
  errorFetchingFriends: null,
  friends: [{ name: 'ivo', id: 3 }, { name: 'goat', id: 4 }]
};

describe('FriendListMenu', () => {
  it('should render component correct', () => {
    const actual = renderer.create(
      <Provider store={createStore(combineReducers({ transactionReducer }))}>
        <FriendListMenu {...props}/>
      </Provider>
    );
    expect(actual).toMatchSnapshot();
  });
});