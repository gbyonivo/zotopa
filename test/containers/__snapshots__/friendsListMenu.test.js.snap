// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FriendListMenu should render component correct 1`] = `
<div>
  loading
</div>
`;
